server/settings.py

```diff
@@ -111,3 +112,5 @@
 
 DEFAULT_URGENCY_VALUE_FOR_MANUAL_ARTICLES = None
 DEFAULT_SOURCE_VALUE_FOR_MANUAL_ARTICLES = 'PA'
+
+ENABLE_PROFILING = True
```

Comment:
or even it remove it from here but read env var from superdesk-core default settings